make check failed on Mac

## Bug Report

<!-- Please answer these questions before submitting your issue. Thanks! -->

make check failed on Mac
### What did you do?

<!-- If possible, provide a recipe for reproducing the error. -->
run command `make check`

### What did you expect to see?
check ok on Mac

### What did you see instead?
```
leakcheck ...
leakcheck ...
xx/pingcap/pd/pkg/cgroup/cgroup_mock_test.go:41:1: test function TestCgroupsGetMemoryUsage is not covered by goleak (goleak not imported)
xx/pingcap/pd/pkg/cgroup/cgroup_mock_test.go:123:1: test function TestCgroupsGetMemoryInactiveFileUsage is not covered by goleak (goleak not imported)
make: *** [static] Error 3
```

### What version of PD are you using (`pd-server -V`)?
master

--- a/.golangci.yml
+++ b/.golangci.yml
@@ -245,6 +245,10 @@ linters:
           disabled: false
           exclude:
             - ""
+          arguments:
+            - [""] # AllowList
+            - [""] # DenyList
+            - - skip-package-name-checks: true
         - name: waitgroup-by-value
           severity: warning
           disabled: false

--- a/pkg/cgroup/cgroup_cpu_test.go
+++ b/pkg/cgroup/cgroup_cpu_test.go
@@ -25,15 +25,9 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
-	"go.uber.org/goleak"
 
 	"github.com/tikv/pd/pkg/errs"
-	"github.com/tikv/pd/pkg/utils/testutil"
 )
-
-func TestMain(m *testing.M) {
-	goleak.VerifyTestMain(m, testutil.LeakOptions...)
-}
 
 func checkKernelVersionNewerThan(re *require.Assertions, t *testing.T, major, minor int) bool {
 	u := syscall.Utsname{}

--- a/pkg/cgroup/cgroup_mock_test.go
+++ b/pkg/cgroup/cgroup_mock_test.go
@@ -22,7 +22,12 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
+	"go.uber.org/goleak"
 )
+
+func TestMain(m *testing.M) {
+	goleak.VerifyTestMain(m)
+}
 
 func isError(err error, re string) bool {
 	if err == nil && re == "" {

